### DISCUSSION TITLE
Manage digests for docker and git refs

### DISCUSSION THREAD

**Discussion by shykes** [dagger/dagger#2084] [Ideas]
Sometimes my configuration fetches a git or docker repository without specifying a digest, and I want to add the current digest to avoid unexpected changes in the future. But how do I find out the current digest? Dagger does not provide a command to do this. I have to search for specialized tooling and learn how to use it, just to get that ref.

Since git and docker refs and digests are important concepts in Dagger, it makes sense for the `dagger` command to make it easy to manage digests.

**Comment by samalba** [comment]
Related to #338, I suggested to have `op: "fetch-container"` to output `/dagger/commit` or similar, that provides the ability to export it as needed. Similar mechanism than "push-container". It's actually implementable without changing the current spec.
